TYPO3.Flow/Classes/TYPO3/Flow/Http/Request.php

```diff
@@ -84,23 +102,15 @@ public function __construct(array $get, array $post, array $files, array $server
         }
         $this->setMethod($method);
 
-        if ($this->headers->has('X-Forwarded-Proto')) {
```

Comment:
Mmm, is that replaced somewhere? I think it's important for vagrant for example.